index.html

```diff
@@ -4832,6 +4832,88 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="mark">
+			<rdef>mark</rdef>
+			<div class="role-description">
+				<p>A mark contains a run of content, marked or highlighted to indicate that other content refers to it, or to indicate special relevance to the user's current activity.</p>
```

Comment:
I'm ok with this. However, with most of the other role-parity* roles, we've incorporated the language from the HTML spec for the corresponding element. Do you think it would be worthwhile to do so here? https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-mark-element

*I realize your needs are not role-parity related. But given that there is a `mark`, the new `mark` role will also be addressing role parity.

Reply:
This works for me. Thanks!

We should raise this at the upcoming ARIA meeting to see if everyone agrees this can land in master.